app/assets/javascripts/discourse/tests/acceptance/bookmarks-test.js

```diff
@@ -25,7 +27,10 @@ acceptance("Bookmarking", function (needs) {
   needs.user();
   let steps = [];
 
-  needs.hooks.beforeEach(() => (steps = []));
+  needs.hooks.beforeEach(function () {
+    KeyboardShortcutInitializer.initialize(getApplication());
+    steps = [];
```

Comment:
where is steps being used ?

Reply:
For https://github.com/discourse/discourse/blob/e0036b6f74667fdcb3bb9cb29bc8e3f5b055e215/app/assets/javascripts/discourse/tests/acceptance/bookmarks-test.js#L48-L48 and https://github.com/discourse/discourse/blob/e0036b6f74667fdcb3bb9cb29bc8e3f5b055e215/app/assets/javascripts/discourse/tests/acceptance/bookmarks-test.js#L90-L96, checking all the reminder type steps have been done on each spec

Reply:
Hmmm weird didn't see it when grepping for it, ok :+1: